File: tests/graph/eval/fixtures/kotlin_spring_sample/src/main/kotlin/com/example/blog/entity/Post.kt
package com.example.blog.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Id
import jakarta.persistence.JoinColumn
import jakarta.persistence.ManyToOne
import jakarta.persistence.Table

@Entity
@Table(name = "posts")
class Post {
    @Id
    @GeneratedValue
    var id: Long? = null

    @ManyToOne
    @JoinColumn(name = "author_id")
    var author: User? = null

    @Column
    var title: String = ""

    @Column(columnDefinition = "TEXT")
    var body: String = ""
}

File: tests/graph/eval/fixtures/fastapi_large/app/db/models/post.py
from sqlalchemy import Column, ForeignKey, Integer, String, Text
from sqlalchemy.orm import relationship

from app.db.session import Base


class Post(Base):
    __tablename__ = "posts"

    id = Column(Integer, primary_key=True, index=True)
    author_id = Column(Integer, ForeignKey("users.id"), nullable=False)
    title = Column(String, nullable=False)
    body = Column(Text, nullable=False)

    author = relationship("User", backref="posts")
